docs/src/pages/[platform]/connected-components/authenticator/NextSteps.tsx

```diff
@@ -0,0 +1,30 @@
+import { CardLink, CardLinkGroup } from '@/components/CardLink';
+import { MdOutlineAutoAwesome, MdTune, MdFlipToFront } from 'react-icons/md';
+import { useCustomRouter } from '@/components/useCustomRouter';
+
+export function NextSteps() {
```

Comment:
I included this as a custom component instead of in the MDX because It's included on the page that is used for all platforms and need platform specific links to work.